src/plugins/music-together/style.css

```
@@ -140,6 +140,123 @@
   text-align: center;
 }
 
+.music-together-diagnostics {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  gap: 12px;
+
+  box-sizing: border-box;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)